src/shapes/textbox.class.js

```diff
@@ -259,45 +259,33 @@
      * to.
      */
     _wrapLine: function(ctx, text, lineIndex) {
-      var maxWidth  = this.width,
-          lineWidth = this._measureText(ctx, text, lineIndex, 0);
```

Comment:
There is no reason to measure the length of the line when measuring single words has more or less same cost, and we are likely doing it anyway.